src/main/java/edu/byu/cs/autograder/PassoffTestGrader.java

```diff
@@ -45,6 +50,11 @@ public PassoffTestGrader(String phaseResources, String netId, String repoUrl, Ob
         super(repoUrl, netId, observer, phase);
         this.stageTestsPath = new File(stagePath + "/tests");
         this.phaseTests = new File(phaseResources);
+        this.module = switch (phase) {
+            case Phase0, Phase1 -> "shared";
+            case Phase3, Phase4 -> "server";
+            case Phase6 -> "client";
```

Comment:
I think Phase 6 grading will need the server jar, but we might also need the client for checking unit tests. We can cross this bridge later though